### PR TITLE
Attempt to debloat Lenovo packages

### PR DESCRIPTION
**Pull request by shuvashish76**
Addresses #6

### CODE REVIEW

File: oem.json [hunk 1]
```diff
@@ -592,6 +592,13 @@
         "description": "Some kind of theme overlay for Nokia devices?\nSome users claim to not see any differences when removed.",
         "removal": "caution"
     },
+    {
+        "id": "com.dti.lenovo.tablet",
+        "label": "Mobile Services",
+        "description": "Silently installs unwanted packages without user permission.\n",
+        "web": "https://www.andrewnile.co.uk/blog/lenovo-mobile-services/.\nPithus report https://beta.pithus.org/report/4ebcc8b5a13851054d85153b0d4086e4b2b5adaee66eaea5843078c6134e9dd7",
```
Comment (MuntashirAkon) [inline oem.json:599]: Web takes array of websites.

e.g.

```json
"web": [
  "https://www.andrewnile.co.uk/blog/lenovo-mobile-services/",
  "https://beta.pithus.org/report/4ebcc8b5a13851054d85153b0d4086e4b2b5adaee66eaea5843078c6134e9dd7"
]
```

Comment (shuvashish76) [reply oem.json:599]: So other texts are not allowed for `web`? ~Pithus report~
Please fix it from your side, I'll follow the same format next time.
Also request to add an [example](https://github.com/0x192/universal-android-debloater/wiki#how-to-contribute) in README for non-technical users like me. Thanks.

Comment (MuntashirAkon) [reply oem.json:599]: > So other texts are not allowed for `web`? ~Pithus report~

No. It already includes pithus.org, adding **Pithus report** is redundant anyway.

**Comment by shuvashish76** [comment]
Some packages missing `description` as no relevant info found. But tested on my device & no major issues detected.

**Comment by MuntashirAkon** [comment]
Descriptions are mandatory. Use something like “Not sure what it does.” This would allow others who might know about it to contribute.

**Pull request merged**